tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -190,6 +178,28 @@ public partial class {loop.ClassName}
         {Each(loop.Headers, header => @"
         private StringValues _" + header.Identifier + ";")}
 
+        {Each(loop.Headers, header => $@"
+        public StringValues Header{header.Identifier}
```

Comment:
To avoid accidentally conflicting any other class members, because the header names are all over the map